src/containers/AccountConnection.jsx

```diff
@@ -322,14 +323,18 @@ class AccountConnection extends Component {
             onForceConnection={forceConnection}
             onSubmit={this.onSubmit}
             submitting={submitting || isRunning}
+            allRequiredFieldsAreFilled={allRequiredFieldsAreFilled}
+            isValid={isValid}
             success={success}
             trigger={trigger}
             closeModal={closeModal}
+            dirty={dirty}
           />
         ) : (
           <KonnectorInstall
             isFetching={isFetching}
             account={account}
+            editing={editing}
```

Comment:
I think that the main goal of `KonnectorInstall` is to never be in an `editing` state, so we should not provide this prop here. Also, it's look more like a typo.